mobile/lib/common/settings_screen.dart

```diff
@@ -1,9 +1,11 @@
 import 'package:flutter/material.dart';
 import 'package:get_10101/features/trade/settings_screen.dart';
 import 'package:get_10101/features/wallet/settings_screen.dart';
+import 'package:get_10101/bridge_generated/bridge_definitions.dart' as bridge;
 
 class SettingsScreen extends StatelessWidget {
-  const SettingsScreen({required this.fromRoute, super.key});
+  final bridge.Config config;
+  const SettingsScreen({required this.fromRoute, super.key, required this.config});
```

Comment:
🔧 The config is registered as provider so instead of passing it through as an argument you could have simple used it like this.

```
final config = context.read<bridge.Config>();
```

Reply:
Nice, thanks for this :)